src/renderer/components/dock/editor-panel.tsx

```diff
@@ -70,17 +79,14 @@ export class EditorPanel extends React.Component<Props> {
 
   onTabChange = () => {
     this.editor.focus();
+    monacoModelsManager.getModel(this.props.tabId).setValue(this.props.value ?? "");
```

Comment:
Nit: extract `monacoModelsManager.getModel(this.props.tabId)` to separate variable to avoid duplicating.